docker-compose.yml

```diff
@@ -36,8 +100,12 @@ services:
       POSTGRES_USER: signals
       POSTGRES_PASSWORD: insecure
     volumes:
-      - "~/.ssh/datapunt.key:/root/.ssh/datapunt.key"
       - postgres-data:/var/lib/postgresql/data
+    healthcheck:
+      test: ["CMD-SHELL", "sh -c 'pg_isready -U signals -d insecure'"]
```

Comment:
This command is not correct, I see the following repeated in my logs:
```
database_1       | 2022-10-24 22:43:48.334 UTC [1177] FATAL:  database "insecure" does not exist
```
The `-d` flag is for database, not password (https://www.postgresql.org/docs/current/app-pg-isready.html).

Reply:
fixed